content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-profiles/diagram.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: blocks/image_content_block
+image: content-strategy/user-profile.svg
+link: content-strategy/user-profile.svg
+reverse: false
+stackPosition: top
+imageAlt: User profile example
+level: 2
+display: 3
+long_description: new-long-description.md
+---
+
+### [3]Main header: (primary user type) profile
```

Comment:
I'm unsure why there are two headings of equal weight here? Maybe one's a placeholder? Nonetheless, please update the YAML so that the partial is used correctly.

This section heading should be added to the YAML at the top of the file as `title: Main header: (primary user type) profile`
The`level` and `display` params in the YAML need to be adjusted to '3' and '2' respectively.

Reply:
Fixed